### ISSUE TITLE
RCS1057 - False Positive - It will try and put a space between two variable declarations in Unity specific code

### ISSUE THREAD

**Issue by kabamjdenton** [JosefPihrt/Roslynator#590]
**Product and Version Used**: 
Latest Roslynator, VSCode, Unity

**Steps to Reproduce**:
This code snippit will reproduce the problem:

	[Header("Header")]
	[SerializeField] private int _int1;
	[SerializeField] private int _int2;

It will suggest putting a space between _int1 and _int2

**Comment by JosefPihrt** [comment]
Hi, could you be more specific on where is the problem. I'm not familiar with Unity. Thanks.

**Comment by Gameghostify** [comment]
I think the problem here is that Roslynator suggests a *newline* between L2 and L3

However, this is not a bug, right? Fields that have any Attribute should have a newline between them and be formatted like 

```cs
[Header("Header")]
[SerializeField]
private int _int1;

[SerializeField]
private int _int2;
```

**Comment by JosefPihrt** [comment]
@Gameghostify yes this behavior is currently by design.

But it makes sense to disable this analyzer by default because it is formatting analyzer.
